Automattic/wp-calypso
# [WordPress → Jetpack app branding] Update 2FA confirmation screen

> **Warning**
> This PR should not be merged until D94684-code has been deployed

_Context:_ Following the plans to [refocus the WordPress apps on core features](https://make.wordpress.org/mobile/2022/07/27/refocusing-the-wordpress-app-on-core-features/), we plan to move features that are specific to WordPress.com to the Jetpack app. As such, we will eventually replace all mentions of the WordPress app in Calypso with Jetpack.

#### Proposed Changes

* This PR mentions of the WordPress app on the 2FA confirmation page with references to the Jetpack app. As part of this, the page's animation has been updated with the Jetpack logo and different colours.   
* The update animation can be played below:

https://user-images.githubusercontent.com/2998162/206732827-9d81bd4d-4275-4b22-aa4c-d4852c9a40a9.mov

#### Testing Instructions

* _Prerequisite:_ A WordPress.com account with 2FA enabled as well as the Jetpack app that's logged into that account.
* With this branch checked out go through the WordPress.com login flow and view the 2FA confirmation page.
* Verify that the updated page displays as it should, with references to the Jetpack app and an updated animation.

| Before  | After |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/2998162/206274819-50ed8963-54ac-4757-b6e2-a984b788c1fd.png" width="100%"> | <img src="https://user-images.githubusercontent.com/2998162/206275412-d53a1937-9918-44d7-b811-2679ea920e76.png" width="100%"> |

#### Pre-merge Checklist
